lib/tlscontext.h

```diff
@@ -44,6 +44,16 @@ typedef enum
   TVM_REQUIRED=0x0020,
 } TLSVerifyMode;
 
+typedef enum
+{
+  TSO_NONE,
+  TSO_NOSSLv2=0x0001,
+  TSO_NOSSLv3=0x0002,
+  TSO_NOTLSv1=0x0004,
+  TSO_NOTLSv11=0x0008,
+  TSO_NOTLSv12=0x0010,
+} TLSSslOptions;
+
```

Comment:
And what if we have a `typedef` for `TlsSslOption` and `static const TlsSslOption ` delarations for the values? Or we just change the return type of `tls_lookup_options` to `int`...

I think the problem with `enum` is that when we return from `tls_lookup_options` we say that type of the value is `TLSSslOptions` but it's not: it`s a combination of the enumerated values.